packages/storefront-lwc/src/modules/commerce/productList/productList.js

```diff
@@ -10,6 +10,13 @@ export default class ProductList extends LightningElement {
     @api products;
     @api sort;
     @api sortRuleValue;
+    @api total;
+    @api offset;
+    @api limit;
+
```

Comment:
Wonder if we should start thinking when to use objects vs primitives. This pagination data seems like a good candidate for encapsulation.